layouts/_default/_markup/render-image.html

```diff
@@ -4,10 +4,10 @@
 {{- if $image }}
     {{- with .Page -}}
         {{- $imageOptions := "" -}}
-        {{- with .Params.convertimagesto -}}
+        {{- with .Params.convertImagesTo -}}
             {{- $imageOptions = printf "%dx%d %s" $image.Width $image.Height . -}}
         {{- else -}}
-            {{- with .Site.Params.convertimagesto -}}
+            {{- with .Site.Params.convertImagesTo -}}
```

Comment:
I thought that using .Params.varable only worked with all lowercase; or did I miss that also being fixed (it used to be the case, and there was a note in the docs about it, but I can't find it now).

Reply:
Oh, cool it works with camelCase now! (I spun up a test).